AdyenDropIn/Modules/Common/Utilities/ActionWrapperViewController.swift

```diff
@@ -4,26 +4,27 @@
 // This file is open source and available under the MIT license. See the LICENSE file for more info.
 //
 
-import Foundation
+import Adyen
 import UIKit
 
 /// A modal wrapper for action view controllers, adds a navigation bar with cancel button and handles dismissal.
 internal class ActionWrapperViewController: UINavigationController {
-    
+
     // MARK: - Properties
 
     private var onCancel: (() -> Void)?
-    
+
     // MARK: - Initializers
 
     internal init(
-        rootViewController: UIViewController,
+        actionComponent: any PresentableComponent,
```

Comment:
Property name is action component but type is `presentableComponent`. We can pass a payment component here too then? What happens in that case?